graviti/manager/base.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Basic methods of some manager classes."""
+
+from typing import Any, Dict
+
+from tensorbay.dataset import Notes, RemoteData
+from tensorbay.label import Catalog
+from tensorbay.utility import URL
+
+from graviti.client import get_catalog, get_notes, list_data_details, list_segments
```

Comment:
We still use the old OpenAPIv1 here, shall we update them to v2?

Reply:
After the openapi is updated, we will push another pr to update the `api` and the `params in Sheets`. Now, Draft can not use `self.access_key`